Userland/Libraries/LibJS/Runtime/Error.cpp

```diff
@@ -14,6 +14,8 @@
 
 namespace JS {
 
+static FlyString unknown_function_name = "<unknown>";
```

Comment:
I don't think this is needed, there will only ever be one such FlyString in memory anyway.

Reply:
It is more because it is used three times, should I inline all of those?

Reply:
Again, the whole point of FlyStrings is that they become deduplicated anyway and all of them will point to the same object in memory. And it's not exactly performance critical code either, this is the error class after all :)

Reply:
I meant from a having `"<unknown>"` in the code three times perspective. But I'll just put the string in the usages.